src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ConflictResolution.targets

```diff
@@ -60,20 +55,26 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)" />
       <ReferenceCopyLocalPaths Include="@(_ReferenceCopyLocalPathsWithoutConflicts)" />
     </ItemGroup>
+
   </Target>
 
-  <Target Name="_HandlePublishFileConflicts" AfterTargets="RunResolvePublishAssemblies"
-          DependsOnTargets="GetReferenceAssemblyPaths">
-    <ResolvePackageFileConflicts ReferenceCopyLocalPaths="@(ResolvedAssembliesToPublish)"
-                                PlatformManifests="@(PackageConflictPlatformManifests)"
-                                TargetFrameworkDirectories="$(TargetFrameworkDirectory)"
-                                PreferredPackages="$(PackageConflictPreferredPackages)">
-      <Output TaskParameter="ReferenceCopyLocalPathsWithoutConflicts" ItemName="_ResolvedAssembliesToPublishWithoutConflicts" />
+  <Target Name="_HandlePackageFileConflictsForPublish"
```

Comment:
Do we still need to do conflict resolution separately for build and publish?  If so, can you add comments explaining why, and what the difference is?